src/shacl-shapes.xsl

```diff
@@ -63,6 +63,7 @@
             <xsl:for-each select="$namespacePrefixes/*:prefixes/*:prefix/@importURI">              
                 <owl:imports rdf:resource="{.}"/>
             </xsl:for-each>      
+            <owl:imports rdf:resource="{$coreArtefactURI}"/>
```

Comment:
looks like we have the correct variable name.
Here however we shall add also the "restrictions layer artefact".

Reply:
added